docs/diagnostics/PX1030.md

```diff
@@ -8,16 +8,20 @@ This document describes the PX1030 diagnostic.
 | PX1030 | The `PXDefault` attribute of the field is used incorrectly. | Warning (Level 1: Significant) or Error | Available | 
 
 ## Diagnostic Description
+For the information on how to use the `PXDefault` attribute correctly, see [Default Values](https://help.acumatica.com/Help?ScreenId=ShowWiki&pageid=59a784a5-7663-4031-ab15-594c2ec1c027).
+
+If the `PXDefault` attribute is used without the `PersistingCheck` property set to `Nothing`, the attribute used on a custom field defined in `PXCacheExtension` can prevent persisting of records to the database.
+
 If a DAC or DAC extension includes an unbound field with the `PXDefault` attribute that doesn't have `PersistingCheck = PXPersistingCheck.Nothing`, an error is displayed. For this error, you can select one of the following two code fixes:
 
  - A code fix that changes the `PXDefault` attribute to `PXUnboundDefault`
  - A code fix that adds `PersistingCheck = PXPersistingCheck.Nothing` to the `PXDefault` attribute
 
 If a DAC extension includes a bound field with the `PXDefault` attribute that doesn't have `PersistingCheck = PXPersistingCheck.Nothing`, a warning is displayed. For this warning, the code fix adds `PersistingCheck = PXPersistingCheck.Nothing` to the `PXDefault` attribute.
 
-If the `PXDefault` attribute is used without the `PersistingCheck` property set to `Nothing`, the attribute used on a custom field defined in `PXCacheExtension` can prevent persisting of records to the database.
+The diagnostic is not displayed for the fields of the DACs that contain only unbound fields.
```

Comment:
I would add a more examples of a fully unbounded DACs - DACs for smart panels or filters

Reply:
Fixed